src/xdotoolify.js

```diff
@@ -86,14 +90,30 @@ var _getElementAndBrowserRect = async function(page, selector) {
   }
 };
 
-var _getElementAndBrowserScreenRect = async function(page, selector) {
+var _getElementAndBrowserScreenRect = async function(
+  page,
+  selector,
+  timeout = 10000,
```

Comment:
Timeout should be optional. If no timeout, then it should throw error if it doesn't exist like before.

Why is starttime an argument here?

This should be a loop instead of a recursive call. babel will automatically turn the loop into a set of recursive calls for you.